src/util/mpz.cpp

```diff
@@ -146,11 +144,6 @@ mpz_manager<SYNCH>::mpz_manager():
     else {
         m_init_cell_capacity = 6;
     }
-    for (unsigned i = 0; i < 2; i++) {
```

Comment:
Seems that we will need to copy over the initialization of m_tmp[I], Only m_args was removed.

for (unsigned i = 0; i < 2; i++) { 
   m_tmp[i] = allocate(m_init_cell_capacity); 
}

Reply:
If this place is OK now?

Reply:
my comment was misguided.